doc/plugins/limit-count.md

```diff
@@ -37,7 +37,7 @@ Limit request rate by a fixed number of requests in a given time window.
 | -------------- | ------- | -------------------- | ------- | ------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | count          | integer | required             |         | [0,...]                                                                  | the specified number of requests threshold.                                                                                                                                                                                                                                                                 |
 | time_window    | integer | required             |         | [0,...]                                                                  | the time window in seconds before the request count is reset.                                                                                                                                                                                                                                               |
-| key            | string  | required             |         | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for"] | the user specified key to limit the rate.                                                                                                                                                                                                                                                                   |
+| key            | string  | required             |         | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for", "consumer_name"] | the user specified key to limit the rate.                                                                                                                                                                                                                                                                   |
```

Comment:
The Description of `key` should be `the user specified key to limit the counts .`

Reply:
done.

Reply:
- Do you mean that I should introduce the function of each key like limit-conn doc? @Firstsawyou
-  I seem to have misunderstood you before.

Reply:
I mean `rate` should be changed to `counts`.

Reply:
OK, I think the explanation of the document is not complete enough, so I added the Chinese and English documents

Reply:
nice